Wikipedia/Code/ArticleViewController.swift

```diff
@@ -57,6 +57,14 @@ class ArticleViewController: ViewController, HintPresenting {
         return rc
     }()
     
+    lazy var referenceWebViewBackgroundTapGestureRecognizer: UITapGestureRecognizer = {
+        let tapGR = UITapGestureRecognizer(target: self, action: #selector(tappedWebViewBackground))
+        tapGR.delegate = self
```

Comment:
```suggestion
```

We don't appear to be leveraging the delegate methods for the gesture recognizer, so we could remove this.

Reply:
Actually without this and the delegate method conformance in `ArticleViewController+References.swift`, `tappedWebViewBackground` never fires so `ReferenceBackLinksViewController` never dismisses upon tap. I did add a gesture recognizer check in 0578c0d before returning true though just to target it a little better.

Reply:
Ahhh – I gotcha. Sorry, I was still misunderstanding the different appearance modes of the references controller (with citation modal on in-article tap, without modal when triggered from citation) and wasn't actually even hitting the `tappedWebViewBackground` code path during testing. This led me to incorrectly believe the delegate conformance was unnecessary.

Looks good!

Reply:
Thanks, no problem! Yeah the different reference view modes are a little confusing.